tests/format/typescript/non-null/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -59,11 +59,11 @@ this.foo.get("bar")!.doThings().more();
 foo!.bar().baz().what();
 
 =====================================output=====================================
-const { somePropThatHasAReallyLongName, anotherPropThatHasALongName } =
-  this.props.imReallySureAboutThis!;
+const { somePropThatHasAReallyLongName, anotherPropThatHasALongName } = this
+  .props.imReallySureAboutThis!;
```

Comment:
I prefer prev version.

Reply:
I updated the already existing logic to check if it is a complex destructuring, simplified it and added a condition to evaluate arrays too. 
So now it goes for "break-lhs" if it is a destructuring assignment and there are more than one element on the left side.

So now for this particular case I have this:

```js
const {
  somePropThatHasAReallyLongName,
  anotherPropThatHasALongName,
} = this.props.imReallySureAboutThis!;
```

Is it fine? And if not, why in this case the destructuring should not break?